src/components/Tag.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { Link } from 'gatsby';
+import styled from '@emotion/styled';
+import { rgba } from 'polished';
+
+const isInteractive = ({ as, interactive }) =>
+  interactive || as === 'a' || as === Link;
+
+const Tag = styled.span`
```

Comment:
Can we use the component you just merged into the `@newrelic/gatsby-theme-newrelic`?

Reply:
Got, saw your comment after I reviewed. I'll handle this for ya